features/rerun_profile.feature

```diff
@@ -26,15 +26,15 @@ Feature: Rerun profile
     And a file named "features/step_definitions/rerun_steps.rb" with:
       """
       Given('fixed now') do
-        log 'All fixed now'
+        $stdout.puts 'All fixed now'
```

Comment:
`log` is not defined in `cucumber` v3 so fall back to something that works in v3, 4 and 5 without deprecation warnings.